modules/Bio/EnsEMBL/Variation/MotifFeatureVariation.pm

```diff
@@ -121,6 +121,29 @@ sub motif_name {
   return $self->{motif_name};
 }
 
+=head2 binding_matrix_stable_id
```

Comment:
Should we deprecate the method `motif_name` then ?

Reply:
No. It is still used.

Reply:
Oups! Sorry I didn't read the content of the method! My bad!